### PR TITLE
Maxx 55788 add samples to how tos

### PR DESCRIPTION
**Pull request by hugow**

### CODE REVIEW

**Review by drewavisautodesk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,133 +15,67 @@ practice to package 3ds Max Python tools with pip, we provide all our examples i
...
-The [install.sh](install.sh) script can be used from bash
-to install the samples in 3ds Max. The script needs to be run from a
-3ds Max installation directory.
-
-### Option B: Install Everything Locally in Two Steps (--user)
-> Note: the steps described here need to be done from a git bash prompt
+## Python How Tos
 
-It is possible to break up the installation in two steps.
+The samples below are translations of [MAXScript How Tos](https://help.autodesk.com/view/3DSMAX/2020/ENU/?guid=GUID-25C9AD58-3665-471E-8B4B-54A094C1D5C9) that
```
Comment (drewavisautodesk) [inline README.md]: Maybe point to the 2021 docs now?

**Comment by donnie-hughes** [comment]
QA: Test comment.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # How To - Practical Examples For Python
 
-![Splash](Splash.png)
+![Splash](/doc/Splash.png)
```
Comment (donnie-hughes) [inline README.md:3]: QA: Test review (on changed file).

**Review by donnie-hughes — COMMENTED** [review]
QA: Finishing review.

**Pull request merged**